docs/source/supergraphs.md

```diff
@@ -74,3 +74,15 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+`rover supergraph compose` utilizes the composition function published in the [harmonizer]() package under the hood. Harmonizer is still under rapid development, and some breaking changes are to be expected. For now, newer versions of Rover emit supergraph schemas that are incompatible with older versions of [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway).
```

Comment:
Guessing at what you intended the link to be here
```suggestion
`rover supergraph compose` utilizes the composition function published in the [harmonizer](https://crates.io/crates/harmonizer) package under the hood. Harmonizer is still under rapid development, and some breaking changes are to be expected. For now, newer versions of Rover emit supergraph schemas that are incompatible with older versions of [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway).
```

Reply:
I think it's a bit funny / won't age well to say "For now...". It sounds like we might resolve that in the future, which we will not. Maybe:

>Current versions of Rover emit supergraph schemas that are incompatible with older [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway) versions in the range `<0.39.0`.